plugins/nf-tower/src/main/io/seqera/tower/plugin/TowerReports.groovy

```diff
@@ -145,7 +145,7 @@ class TowerReports {
                     final pattern = patterns.get(p)
                     writer.send { PrintWriter it -> it.println("${pattern}\t${dst}\t${destination.size()}") }
                     final numRep = totalReports.incrementAndGet()
-                    log.debug("Adding report [{}] {} -- {} -- {} -- {} -- {} -- {}", numRep, pattern, dst, destination.getScheme(), destination.toUri(), destination, FilesEx.toUriString(destination))
+                    log.debug("Adding report [{}] {} -- {}", numRep, pattern, dst)
```

Comment:
Provided this is OK. Is it not more idiomatic to use groovy string interpolation instead of placeholders ?

Reply:
I think that to use placeholders is better at logging. This avoids to interpolate the string if you are not going to report that log, otherwise the string interpolation is always computed.

Reply:
AFAIK when using @Slf4j annotation groovy automatically wraps the log statement with a `if` level e.g 

```
if( log.isDebugLevel() ) {
  log.debug "etc"
}
```

Reply:
Yes, you're right. Nice! I'll use groovy string interpolation then.